.changeset/dirty-moles-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/autocomplete": minor
+---
+
+added state.open() and removed state.close() from onPress prop of clear button so that dropdown does not close when clear button is clicked
```

Comment:
I think the version should be a patch. The description should focus on the behavior, not the implementation details.

Reply:
is this okay?
![image](https://github.com/user-attachments/assets/4f50082b-d27a-479c-ae02-f72d5e2b4106)